modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1138 @@
+//#TINY-9268
+
+In previous versions of {productname}, it was not possible to change the `text` or `icon` of a toolbar or menu button dynamically.
+
+In {productname} 6.4.1, two new methods `setText` and `setIcon` were added to the menu and toolbar button APIs, to enable updating the `text` and `icon` of the buttons after the editor initialization.
+
+=== New `highlight_on_focus` option which enables highlighting the content area on focus
+//#TINY-9277
+
+In previous versions of {productname}, the highlighted border around the editor content window did not render when the editor was in focus.
+
+As a consequence, this issue, whilst creating no impending impacts on the functionality of the editor, created issues for users that rely on accessibility features such as this.
+
+To fix this issue, {productname} 6.4.1, when the editor content window is in focus, the editor will render a visible `highlighted` border around the content window when the option is added to the `tinymce.init()` config as below.
+
+.Setting `highlight_on_focus: true` option to tinymce.init().
+[source, js]
+----
+tinymce.init({
+    selector: "textarea",
+    highlight_on_focus: true
+});
+----
+
+
+=== New fontsizeinput toolbar item which allows the user to set the size via input and also increase and decrease it with + and - buttons
+//#TINY-9429
+
+In previous versions of {productname}, users did not have the ability to set their font size by either direct input into a component field or by using plus `+` or minus `-` buttons.
+
+As a consequence, {productname} introduced a new UI component that gives users an option to set the font sizes via the `fontsizeinput` toolbar button. For example:
+
+[source,js]
+----
+tinymce.init({
+    selector: "textarea",  // change this value according to your HTML
+    toolbar: "fontsizeinput",
+    font_size_input_default_unit: "pt"
+});
+----
+
+[NOTE]
+More details about this component and the possible values that can be used within the new `fontsizeinput` input field can be found in `xref:available-toolbar-buttons.adoc#the-core-toolbar-buttons[fontsizeinput]`.
+
+=== Added skipFocus option to the ToggleToolbarDrawer command to preserve focus
+//#TINY-9337
+
+A focusing event that was in the initial *FloatingToolbarButton* config, in previous versions of {productname} in (2019) was causing `focus` issues with the *ToggleToolbarDrawer*. As a consequence, this meant that while the editor was scrolled into `focus`(viewport position), and the *ToggleToolbarDrawer* command `skipFocus: boolean` was set, the toolbar draw could still take `focus` when it was not meant to.
+
+In {productname} 6.4.1, an improvement was made to the focusing `event` which is now removed after toggling the toolbar.
+
+As a result to the fix, while the *FloatingToolbarButton* `focus` is initiated, the toolbar now focuses correctly without stealing `focus` when `skipFocus: 'boolean' is set.
```

Comment:
That was a decent job of describing the code change, but those names do not exist on our API and have no meaning to customers. Here's what changed from an integrator perspective:

```suggestion
In previous versions of {productname}, the *ToggleToolbarDrawer* command contained an incorrect focus call. As a consequence, this meant that when the *ToggleToolbarDrawer* command option `skip_focus: true` was set the toolbar drawer would take `focus` when it was not meant to.

In {productname} 6.4.1, an improvement was made to the focusing code which now checks for `skip_focus`.

As a result of the fix, when using the *ToggleToolbarDrawer* command, the toolbar now avoids stealing focus when `skip_focus: true` is set.
```